types/intersects/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
+        "esModuleInterop": true,
```

Comment:
Could you use something like `import circlePoint = require("./circle-point");
export { circlePoint };` instead of `export { default as circlePoint } from './circle-point';` in `index.d.ts`? I know it can look less nice, but it would allow you to disable `esModuleInterop` in the settings here.

On DT we prefer to follow a default set of compiler options as much as possible, to avoid problems for consumers of the type definitions.